parser.cpp

```diff
@@ -1608,6 +1608,10 @@ namespace Sass {
   {
     lex< identifier >();
     string name(lexed);
+    if (name == "compact") {
```

Comment:
I also saw that, but wasn't sure how to better address it. But we surely can move that to some more appropriate place. IMO we might be able to add a specific if clause just before internal functions are resolved, and if the name is `compact` emit this warning!?